.gitignore

```diff
@@ -9,4 +9,4 @@
 .idea
 *.iml
 
-/test_run_dir/
+/test_run_dir_pre_1172/
```

Comment:
uh wut

Reply:
That's weird, it must have somehow caught a re-name i did on that dir to save some intermediate firrtl file. Undid it.